### PR TITLE
chore: Record npm install time

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskRunNpmInstall.java [hunk 1]
```diff
@@ -249,6 +252,7 @@ private boolean isVaadinHashUpdated() {
      * `package.json` has been updated.
      */
     private void runNpmInstall() throws ExecutionFailedException {
+        long startTime = System.currentTimeMillis();
         // Do possible cleaning before generating any new files.
         cleanUp();
```
Comment (caalador) [inline flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskRunNpmInstall.java:257]: cleanUp might take some time if node_modules is deemed to be removed due to npm->pnpm or pnpm->npm change.
Should it really be taken into account for (p)npm install time?

Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskRunNpmInstall.java:257]: We should definitely record the full time that we need for getting the frontend in shape. But maybe we should record it separately then?

Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskRunNpmInstall.java]: Fixed

Comment (caalador) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskRunNpmInstall.java]: You forgot to push the StatisticsConstants with EVENT_PACKAGEMANAGER_CLEANUP_TIME_PREFIX

Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskRunNpmInstall.java]: I indeed did..

**Pull request merged**